### PR TITLE
UIC Spring 2022 faculty update

### PR DESCRIPTION
**Pull request by kaytwo**
# Contributing to CSrankings

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

Inna Partin-Vaisband is tenure track faculty in ECE with an adjunct appointment in CS who can solely advise PhD students in Computer Science. [Adjunct Faculty List](https://cs.uic.edu/faculty-staff/adjunct-faculty/). Nikita Soni is tenure track in CS. We also removed three faculty who no longer meet the inclusion criteria.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings-i.csv [hunk 1]
```diff
@@ -210,6 +210,7 @@ Ingrid Wetzel,University of Hamburg,https://www.inf.uni-hamburg.de/de/inst/ab/it
 Ingrid Zukerman,Monash University,http://users.monash.edu/~ingrid,DFyOHW4AAAAJ
 Inhwan Lee,Hanyang University,http://csl.hanyang.ac.kr,NOSCHOLARPAGE
 Inkyoung Hur,Nova Southeastern University,https://cec.nova.edu/faculty/Hur-Inkyoung.html,NOSCHOLARPAGE
+Inna Partin-Vaisband,University of Illinois at Chicago,https://ece.uic.edu/profiles/inna-partin-vaisband-phd/,visXuWkAAAAJ
```
Comment (emeryberger) [inline csrankings-i.csv:213]: ECE, not CS, not clear how they meet inclusion criteria.

File: csrankings-i.csv [hunk 2]
```diff
@@ -303,7 +304,6 @@ Isaac Lagaris,University of Ioannina,https://cs.uoi.gr/~lagaris,I7VicCIAAAAJ
 Isaac Triguero,University of Nottingham,https://www.nottingham.ac.uk/computerscience/people/i.triguerovelazquez,KogUP4YAAAAJ
 Isaac Woungang,Ryerson University,https://www.scs.ryerson.ca/~iwoungan,NOSCHOLARPAGE
 Isabel C. Teixeira,Universidade de Lisboa,https://ciencias.ulisboa.pt/en/perfil/sacmadeira,GuI5uHoAAAAJ
-Isabel F. Cruz,University of Illinois at Chicago,https://www.cs.uic.edu/Cruz,kdv2ajAAAAAJ
```
Comment (emeryberger) [inline csrankings-i.csv:306]: Please move to `old/rip.csv`.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-i.csv (hunk 1 above)
Comment (kaytwo) [reply csrankings-i.csv:213]: Dr. Partin-Vaisband is adjunct faculty in CS - https://cs.uic.edu/faculty-staff/adjunct-faculty/.

**Review by emeryberger — COMMENTED** [review]
LGTM

**Pull request merged**